pxr/base/tf/denseHashMap.h

```diff
@@ -56,7 +54,13 @@ template <
     class    EqualKey  = std::equal_to<Key>,
     unsigned Threshold = 128
 >
-class TfDenseHashMap
+
```

Comment:
Okay, we don't want to do this this way. The semantics of TfDenseHashMap now BEING a HashFn and an EqualKey is not desirable.
What I think you really want to do is reimplement the internal member class _VectorHashFnEqualFn using empty base optimization explicitly instead of the compressed_pair